onyo/utils.py

```diff
@@ -190,72 +190,97 @@ def check_float(value):
         return False
 
 
-def edit_file(file, onyo_root, onyo_new=False):
+def create_onyo_temp_file(file, onyo_root):
     """
-    Check if the selected editor exists. Check if a file exists, create a copy
-    of it in .onyo/temp/ to change with the editor, and check the temp file
-    after editing for correct yaml syntax and validity specified by the onyo
-    validation file.
-    If the asset is not valid, open a dialog to enable to correct the asset, or
-    remove the changes.
-
-    Returns True on success.
+    Decide for an existing asset file, if a temporary file needs to be created,
+    or if one already exists and should be used.
+    If no temporary file exists, create a copy of the given asset.
```

Comment:
Why would a temp file already exist? And wouldn't it be dangerous to use an unknown stale temp file that may have no relation to the file that the user asked to edit?